docs/projects/ansible/collection/centurion/playbooks/teams.md

```diff
@@ -0,0 +1,68 @@
+---
+title: Teams
+description: Creation and patching of Centurion ERP teams and permissions 
```

Comment:
```suggestion
description: Ansible Playbook for Creating and patching of Centurion ERP teams, including permissions.
```

Reply:
fixed in commit [054c201](https://github.com/nofusscomputing/ansible_collection_centurion/pull/17/commits/054c201d1abd6accf7ee94843a472849075bd5e1)